Bigtable/src/Instance.php

```diff
@@ -292,11 +275,57 @@ public function create(array $clusterMetadataList, array $options = [])
                 'labels' => $labels
             ],
             'clusters' => $clustersArray
-        ]);
+        ]+ $options);
 
         return $this->resumeOperation($operation['name'], $operation);
     }
 
+    /**
+     * This method returns an associative array containing data required for
+     * configuring Bigtable instance clusters. The resulting array will contain
+     * 'clusterId', 'locationId`, and depending on how it is called,
+     * `storageType` and `serveNodes`.
+     *
+     * Example:
+     * ```
+     * $cluster = Instance::buildClusterMetadata('my-cluster', 'us-east1-b');
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @param string $clusterId The cluster ID
+     *        e.g., just `cluster-id` rather than `projects/project-id/instances/instance-id/clusters/cluster-id`.
+     * @param string $locationId The location ID
+     *        e.g., just `us-east1-b` rather than `projects/project-id/locations/us-east1-b`.
+     * @param int $storageType The storage media type for persisting Bigtable data. Possible values include
+     *        `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_SSD`,
+     *        `Google\Cloud\Bigtable\Instance::STORAGE_TYPE_HDD`.
+     *        **Defaults to** @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#google.bigtable.admin.v2.StorageType StorageType
+     * @param int $serveNodes The number of nodes allocated to this cluster.
+     *        More nodes enable higher throughput and more consistent performance.
+     * }
+     *
+     * @return array
+     * @codingStandardsIgnoreEnd
+     */
+    public static function buildClusterMetadata(
+        $clusterId,
+        $locationId,
+        $storageType = null,
```

Comment:
Shouldn't it default to whatever you want, but not null?

Reply:
no, we want to let protobuf make it default as per its specification, thats why pointed to specification in default.